charm/config.yaml

```diff
@@ -7,3 +7,7 @@ options:
     default: 10
     description: Number of seconds to wait for keepalive connections
     type: int
+  api_port:
```

Comment:
As in the previous PR, take this out - we don't need a config option for it because there's no reason for it to ever change.